tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
-envlist = lint,py27,pypy,py34,py35,py36,py37,pypy3,docs
+envlist = lint-{code-style,dist-meta,mypy},docs,py37,py27,pypy3,py36,py35,py34,pypy
```

Comment:
I don't think running separate checks locally is kind to users or their experience. We should stick with `lint` here otherwise we're creating 3 new virtualenvs and incurring the overhead of creating/using them each time. Let's be nice to our contributors and not discourage them from running the test suite.

Reply:
Good point. OTOH we could explicitly point envs to use the same venv.

Reply:
Fixed